src/vg/civcraft/mc/citadel/reinforcement/PlayerReinforcement.java

```diff
@@ -70,6 +78,15 @@ public boolean isBypassable(Player p){
 		return gp.isAccessible(type, PermissionType.BLOCKS);
 	}
 	
+	public int getDamageMultiplier(){
+		Timestamp ts = gm.getTimestamp(g.getName());
```

Comment:
I'd recommend the following:

Don't store a reference to GM. Get a fresh one as here: https://github.com/Civcraft/Citadel/pull/166/files#diff-dbd016df79694880ff7b2e0bc5870217R157

On that same line note that the standard is now pass the Group object to the Manager call. Make the change here and in Namelayer; don't pass the Name back, just the group object and let Namelayer GM pick what data from the group object to use for lookups.